### ISSUE TITLE
Fonts don't load outside VSCode - Probably not for other users either.

### ISSUE THREAD

**Issue by Izhaki** [excalidraw/excalidraw-vscode#86]
Could be a regression caused by https://github.com/excalidraw/excalidraw-vscode/issues/60.

When you save a drawing into, say, `test-01.excalidraw.svg` the svg will have:

```
      @font-face {
        font-family: "Virgil";
        src: url("https://file%2B.vscode-resource.vscode-cdn.net/Users/john.smith/.vscode/extensions/pomdtr.excalidraw-editor-3.5.1/public//dist/excalidraw-assets/Virgil.woff2");
      }
```

The font will not load anywhere outside VScode.  I suspect it wouldn't even load on other machines using VSCode (as the URL includes the username).

# User Story

As a user, I'd like to save my files using the `.excalidraw.svg` extension so:

- I can commit these into the repo for others to contribute.
- I can reference them from `*.md` file (for documentation portals) so they load both locally and once deployed.

In other words, would help if these URLs point to some public URL.

**Comment by pomdtr** [comment]
Thanks for notifying me, this is definitely linked with #60. I'll talk with the excalidraw team about the best way to solve this.

**Comment by sransara** [comment]
Falling back to using `.excalidraw.png` for now. A solution for this issue with `svg` would be much appreciated.

**Comment by pomdtr** [comment]
Since the latest version of excalidraw, the svg fonts are embedded in svgs
